Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -455,7 +455,15 @@ protected override void OnLoad (EventArgs e)
       {
         var columns = EnsureColumnsGot();
         EnsureEditModeRestored();
-        EnsureRowMenusInitialized();
+
+        var postBackCollection = Context!.Request.Form;
+        var targetInfo = postBackCollection["__EVENTTARGET"];
+
+        if (targetInfo != null && TryGetRowMenuId(targetInfo, out var row))
+        {
+          EnsureRowMenuInitialized(row);
+        }
+
         EnsureCustomColumnsInitialized(columns);
```

Comment:
Please check, I think it should be okay to reorder these so the EnsureCustomColumns is before the complicated custom code.